src/content/docs/ko/guides/integrations-guide.mdx

```diff
@@ -6,11 +6,11 @@ import IntegrationsNav from '~/components/IntegrationsNav.astro'
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 import { Steps } from '@astrojs/starlight/components'
 
-**Astro 통합**은 단 몇 줄의 코드만으로 프로젝트에 새로운 기능과 동작을 추가합니다. 사용자 정의 통합을 직접 작성하거나, 공식 통합을 사용하거나, 커뮤니티에서 구축한 통합을 사용할 수 있습니다.
+**Astro 통합**은 단 몇 줄의 코드만으로 프로젝트에 새로운 기능과 동작을 추가합니다. 공식 통합, [커뮤니티 통합](#더-많은-통합-찾기) 또는 심지어 [사용자 정의 통합을 구축](#나만의-통합-구축)할 수도 있습니다.
 
 통합은 다음을 수행할 수 있습니다.
 
-- React, Vue, Svelte, Solid 및 기타 널리 사용되는 UI 프레임워크를 사용합니다.
+- [렌더러](/ko/guides/framework-components/)를 사용하여 React, Vue, Svelte, Solid 및 다른 인기 있는 UI 프레임워크를 잠금 해제합니다.
 - 몇 줄의 코드만으로 Tailwind 및 Partytown과 같은 도구를 추가합니다.
```

Comment:
I believe we're missing an item here, there's an "enable SSR" item that was added in this commit, see: https://github.com/withastro/docs/pull/8575/files#diff-3070344990142deafc4c3d25796542fb1c86c081a3c2df2797afb28b4695a006R16

Reply:
Problem solved! Thank you for the good point.